README.md

```diff
@@ -1,15 +1,15 @@
 <table style="width: 100%; border-style: none;"><tr>
 <td width="140px" style="text-align: center;"><img src="websphere_icon.png" style="max-width:100%" /></td>
-<td><strong>Visual Studio Team Services Extension for IBM WebSphere Application Server</strong><br />
+<td><strong>Azure DevOps Extension for IBM WebSphere Application Server</strong><br />
 <i>Provides deployment task that automatically installs/updates enterprise applications in WebSphere Application Servers.</i><br />
 <a href="https://marketplace.visualstudio.com/items/ms-vsts.ibm-websphere">Install now!</a>
 </td>
 </tr></table>
 
-# Visual Studio Team Services Extension for IBM WebSphere Application Server
+# Azure DevOps for IBM WebSphere Application Server

```

Comment:
Seems the word 'Extension' is removed by mistake.

Reply:
Apologies for the mistake, thanks for bringing it to my attention. I have updated the file now.